### PR TITLE
Gk 138 dodac komponent z zawodnikami do widoku organizatora

### PR DESCRIPTION
**Pull request by KarolZajac**
- [x] Added Fighters component.
- [x] Navigating between add/delete fighter components.
- [x] Some refactoring.

TODO: create endpoint for deleting fighters.

### CODE REVIEW

File: frontend/components/RegisterPlayerForm.js [hunk 1]
```diff
@@ -20,15 +21,16 @@ const RegisterPlayerForm = () =>{
         surname: '',
         age: '',
         weight: '',
-        option1: '',
-        option2: '',
-        option3: ''
+        // option1: '',
+        // option2: '',
+        // option3: ''
```
Comment (t0maboro) [inline frontend/components/RegisterPlayerForm.js:26]: Imo we can get rid of it here.

File: frontend/components/RegisterPlayerForm.js [hunk 2]
```diff
@@ -43,36 +45,43 @@ const RegisterPlayerForm = () =>{
...
-            py={verticalPadding}
+            <Radio.Group
+                name="playerGender"
+                accessibilityLabel="player gender"
+                value={gender}
+                onChange={value => { setGender(value) }}
+                flexDirection="row"
+                justifyContent="space-between"
+                px={horizontalPadding}
+                py={verticalPadding}
```
Comment (t0maboro) [inline frontend/components/RegisterPlayerForm.js:84]: For now that radio group looks unused, so probably we can remove it.

**Review by t0maboro — APPROVED** [review]
Looks good! We can discuss if we want to keep switches with options, because now we don't use them anywhere - comment just for keeping consistency.  Anyway, it can be merged in that form, because it works well. Removing unused code we can save for the future.

**Comment by KarolZajac** [comment]
For now, I will leave the implementation of other options in our data form - just in case it will be helpful when adding some criteria defining if player can take part in certain tournament. I am merging.

**Pull request merged**